GradeManager/GradeConsole.swift

```diff
@@ -22,6 +22,30 @@ struct GradeConsole {
         }
         return menu
     }
+    
+    func getAddGradeInput() throws -> (String, String, Grade) {
+        guard let input = readLine(), validateStudentGradeInputForAddition(input) else {
```

Comment:
```suggestion
        guard let input = readLine(), 
                   validateStudentGradeInputForAddition(input) else {
```
너무 길면 이렇게 줄바꿈을 하는 것도 가독성을 올리는 방법 중 하나입니다☺️